changelog/next_version.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ueo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Why not just

```bash
DIR=`dirname ${BASH_SOURCE[0]}`
```

Reply:
`BASH_SOURCE` might be relative or having relative symlinks. See also https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within for a full discussion.